style.css

```diff
@@ -0,0 +1,844 @@
+  width: 170px;
+  z-index: 3;
+}
+
+#main-nav .nav-with-image {
+  height: 70px;
+  width: 80%;
+  margin: 0 auto;
+  background-color: #f5f5f5;
+  display: none;
+}
+
+.logout-nav {
+  background-color: #272a31;
+  height: 30px;
+  display: none;
+}
+
+.logout-nav nav {
+  width: 80%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+#main-nav .nav-with-image img {
+  width: 50px;
+  padding-top: 10px;
+}
+
+.nav-with-image li .current {
+  color: #ec5242;
+}
+
+#main-nav .nav-with-image i {
+  margin-top: 70px;
+}
+
+#header_section {
+  background-image:
+    linear-gradient(
+      rgba(255, 255, 255, 0.7),
+      rgba(255, 190, 188, 0.664)
+    ),
+    url(img/headline-2.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: auto;
+  padding-bottom: 20px;
+}
+
+.festival_header,
+.about_header_section .festival_header {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.about_header_section .welcome,
+#header_section .welcome {
+  color: #ec5242;
+  font-weight: normal;
+  font-size: 30px;
+  padding-top: 60px;
+}
+
+.about_header_section .title,
+#header_section .title {
+  color: #e8240e;
+  text-transform: uppercase;
+  text-shadow: 3px 2px 2px rgba(225, 88, 88, 0.267);
+  font-size: 40px;
+  font-weight: bolder;
+  line-height: 1.2;
+  padding: 10px 0;
+}
+
+.about_header_section .title-1,
+#header_section .title-1 {
+  color: #e8240e;
+  text-shadow: 3px 2px 2px rgba(225, 88, 88, 0.267);
+  font-size: 20px;
+  font-weight: bolder;
+  line-height: 1.2;
+  padding: 10px 0;
+}
+
+.about_header_section .introduction {
+  background-color: #fff;
+  border-radius: 0;
+  padding: 10px;
+}
+
+#header_section .introduction {
+  border: 2px solid rgba(255, 255, 255, 0.25);
+  padding: 10px;
+  color: #333;
+  font-weight: bold;
+  margin: 40px 0 20px 0;
+  border-radius: 10px;
+}
+
+.time_venu {
+  line-height: 2;
+  color: #333;
+}
+
+.time_venu h2 {
+  font-family: 'cocogoose', sans-serif;
+}
+
+.time_venu h3 {
+  line-height: 1.2;
+}
+
+#project_schedule,
+#about_project_schedule {
+  width: 100%;
+  height: 100%;
+  background-color: #272a31;
+  padding-top: 30px;
+}
+
+#project_schedule {
+  background-image: url(./images/program-x.png);
+}
+
+#project_schedule .main-program,
+#wesing_guests .main-program,
+#about_project_schedule .main-program,
+#previous_events .main-program,
+#festival_partners .main-program {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  line-height: 2.5;
+}
+
+.main-program h3 {
+  color: #fff;
+  font-size: 22px;
+  font-family: 'cocogoose', sans-serif;
+}
+
+#project_schedule hr,
+#wesing_guests hr,
+#about_project_schedule hr,
+#previous_events hr,
+#festival_partners hr {
+  border: 0;
+  border-top: 1px solid #ec5242;
+}
+
+.festival_programs,
+.guest_list {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 15px;
+  width: 100%;
+  margin-top: 25px;
+}
```

Comment:
- [ ] It is required to stick to the guiding design as much as possible. This includes, font-size, font-style, font-family, color, background-color, margin, padding among others. The text beside feature speakers are too big as oppose to what's in the guiding template. Kindly reduce the font-size and the line-height of those texts so as to match the giving design. Take a look at the both designs and work your design alongside the sample design.

### Your Design
![speakers-mobile](https://user-images.githubusercontent.com/26597432/219093218-3ecd9427-a4b7-4a1a-90cd-8d12ab4d3c5a.png)

### Guiding Design
![temp-speakers](https://user-images.githubusercontent.com/26597432/219093551-588bd152-9bda-4997-873e-12ad5af52ec5.png)